students/hyein/users/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+# python built-in module
+import json
+import re
 
-# Create your views here.
+# 외부 module
+from django.http import JsonResponse
+from django.views import View
+
+# 내부 module
+from users.models import User
+class SignupView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body) #json -> python dictionary
+            
+            EMAIL_VALIDATION    = r'^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+            PASSWORD_VALIDATION = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&^])[A-Za-z\d$@$!%*#?&]{8,}$'
+            
+            email    = data['email']
+            password = data['password']
+            
+            if not re.match(EMAIL_VALIDATION, email): 
+                return JsonResponse({'Message' : 'Invalid Email'}, status = 400)
+            
+            if not re.match(PASSWORD_VALIDATION, password): 
+                return JsonResponse({'Message' : 'Invalid Password'},status = 400)
+
+            if User.objects.filter(email = email).exists(): 
+                return JsonResponse({'Message' : 'Email already in use'}, status = 400)
+    
+            User.objects.create(
+                name         = data['name'],
+                email        = email,
+                password     = password,
+                phone_number = data['phone_number'],
+                )
+            
+            return JsonResponse({'Message': 'CREATED'}, status=201)
+        
+        except: 
+            return JsonResponse({'Message': 'KEY_ERROR'}, status=400)
```

Comment:
```suggestion
        except KeyError: 
            return JsonResponse({'Message': 'KEY_ERROR'}, status=400)
```

작성해주신건 try 안에서 일어나는 모든 예외 상황에 대해 'KEY_ERROR '라는 메세지를 반환하겠다는 코드입니다.
의도에 맞게 KeyError 만을 잡아주도록 수정해주세요!

Reply:
수정했습니다!